src/plane.js

```diff
@@ -0,0 +1,11 @@
+class Plane {
+    constructor(id) {
+        if (typeof id === 'string' && id.trim() !== '') {
+            this.id = id;
+        } else {
+            throw new Error('Invalid ID');
+        }
+    }
```

Comment:
Even better if

There was a getter method for the id property to allow other parts of the code to access the ID without directly accessing the property itself.